### ISSUE TITLE
Summon-able Creatures

### ISSUE THREAD

**Issue by taelon875** [zunath/CyberpunkZombieSurvival_JVM#171]
Enhancement Spells to either summon pets, or animate existing weapons to fight for them.

**Comment by Drakaden** [comment]
Limit to 1 summon per caster to not disbalance things, melee weapons only, power of summon is dependant on the weapon used for the summon, summon cannot be destroyed by zombies, but have a time limit on it (like 1 minute before the weapon goes back to owner)

**Comment by taelon875** [comment]
How I imagine it working is you cast the spell on the weapon ( I agree melee only ) and that would remove the weapon  from your inventory, the strength is always just going to be the weapon damage as if a player were swinging it with 10 strength. Enhancement skill would determine time lasted. And every 5 Points given via gear could add another summon limit. Upon expiration the summon would be killed and would drop the weapon on the ground in its loot.

**Comment by Drakaden** [comment]
I would disagree with raising the summon limit, it would provide too much baiting, though if you add an hp pool to those weapons, then it's agreeable.

**Comment by taelon875** [comment]
5 points via gear is a lot,  Is this not the normal points that you can put it. I would say it would be a fair balance since it costs them  the weapons they use if they can't retrieve them.

**Comment by Drakaden** [comment]
I'm not sure what you contest in this statement.

**Comment by taelon875** [comment]
Sorry im dead tired. I will edit it to make sense tomorrow when I can think.
